ocaml/idl/datamodel.ml

```diff
@@ -7732,6 +7732,124 @@ module VUSB = struct
       ~messages:[create; unplug; destroy] ()
 end
 
+module Tracing = struct
```

Comment:
Please move this module to a separate file (`ocaml/idl/datamodel/datamodel_tracing.ml`)

Reply:
Does it make more sense to call the class `Tracer` actually? The classes are usually names of "things", like "Host", "VM" etc.